client/blocks/upload-image/style.scss

```diff
@@ -56,3 +56,82 @@
 	top: 50%;
 	transform: translate( -50%, -50% );
 }
+
+.upload-image__image-container {
+	width: 200px;
+	height: 200px;
+
+	&.is-uploaded, &.is-uploading {
+		border: 1px solid lighten( $gray, 10% );
+		background-color: lighten( $gray, 30% );
+	}
+}
+
+.upload-image__uploading-container, .upload-image__uploading-done-container {
+	height: 100%;
+	text-align: center;
+}
+
+.upload-image__uploading-done-container {
+	position: relative;
+}
+
+.button.is-compact.upload-image__uploaded-image-remove {
+	position: absolute;
+		top: 0;
+		right: 0;
+	width: 24px;
+	height: 24px;
+	transform: translate(50%, -50%);
+	border: 1px solid lighten( $gray, 10% );
+	border-radius: 50%;
+	background-color: $gray-light;
+	cursor: pointer;
+	z-index: z-index( '.dialog__backdrop', '.upload-image__uploaded-image-remove' );
+}
+
+.button.is-compact .gridicon.upload-image__uploaded-image-remove-icon {
+	position: absolute;
+		top: 50%;
+		left: 50%;
+	transform: translate(-50%, -50%);
+	width: 20px;
+	height: 20px;
+	margin: 0;
+	color: lighten( $gray, 10% );
+}
+
+.upload-image__uploading-image, .upload-image__uploaded-image-wrapper {
+	position: relative;
+	top: 50%;
+	transform: perspective(1px) translateY(-50%);
```

Comment:
What does the `perspective` achieve here? It's just a 2D transform, not 3D, is it?